test/e2e/config/ck8s-docker.yaml

```diff
@@ -84,8 +85,8 @@ providers:
 
 variables:
   KUBERNETES_VERSION_MANAGEMENT: "v1.28.0"
-  KUBERNETES_VERSION: "v1.30.0"
-  KUBERNETES_VERSION_UPGRADE_TO: "v1.30.1"
+  KUBERNETES_VERSION: "v1.29.6"
+  KUBERNETES_VERSION_UPGRADE_TO: "v1.30.3"
   IP_FAMILY: "IPv4"
```

Comment:
especially when we already repeat the configs here. i can already see confusion about "i updated this value, why does it not work as i expect?"